src/pages/gen2/reference/cli-commands/index.mdx

```diff
@@ -215,9 +215,81 @@ npx amplify generate config --stack amplify-nextamplifygen2-josef-sandbox-ca85e1
 
 ## npx amplify generate graphql-client-code
 
-Generate GraphQL statements for your frontend application to consume.
+Generate GraphQL statements and types for your frontend application to consume. 
+
+<SubCommandHeading parentCommand="amplify-graphql-client-code">Options</SubCommandHeading>
+
+The available parameters for `npx amplify generate graphql-client-code` are:
+
+Required parameters:
+- Stack identifier 
+  - `--stack`: A stack name that contains an Amplify backend.
+- Project identifier
+  - `--app-id`(String) - The Amplify App ID of the project.
+  - `--branch`(String) - A git branch of the Amplify project.
+
+Optional parameters:
+- `--out`(string) - Specifies the path to the directory where the config is written. If not provided, defaults to the current process working directory.
+- `--format`(string (choices: "modelgen", "graphql-codegen", "introspection")) - Specifies the format of the GraphQL client code to be generated.
+- `--model-target` (string (choices: "java", "swift", "javascript", "typescript", "dart")) - Specifies the modelgen export target. Only applies when the --format parameter is set to modelgen.
+- `--statement-target`(string (choices: "javascript", "graphql", "flow", "typescript", "angular")) - Specifies the graphql-codegen statement export target. Only applies when the --format parameter is set to graphql-codegen.
+- `--type-target`(string (choices: "json", "swift", "typescript", "flow", "scala", "flow-modern", "angular")) - Specifies the optional graphql-codegen type export target. Only applies when the --format parameter is set to graphql-codegen.
+- `--all`(boolean)- Shows hidden options.
+- `--profile`(string) - Specifies an AWS profile name.
+- `--debug` (boolean) - Print debug logs to the console.
+- `--help`(boolean) - Displays help information about the command.
```

Comment:
can this be updated to use the same structure and nomenclature of the other commands?